pro/src/new_components/VenueForm/Informations/SiretOrCommentFields/SiretOrCommentFields.tsx

```diff
@@ -43,6 +45,7 @@ const SiretOrCommentFields = ({
   const formatSiret = async (siret: string): Promise<void> => {
     // remove character when when it's not a number
     // this way we're sure that this field only accept number
+    /* istanbul ignore next: DEBT, TO FIX */
```

Comment:
c'est ok de mettre des ignore sur du code existant, ceci dit quand le test est simple à écrire et que ça ne ralentit pas trop le travail, il ne faut pas hésiter à le rajouter !

Reply:
Il faudrait peut-être mettre au clair quand c'est possible de mettre le commentaire ou pas, dans ce cas. J'avais compris que c'était géré pendant des itérations "techniques".

Reply:
J'ai rajouté les tests.